src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -10,8 +10,6 @@ public sealed partial class ProcessStartInfo
     {
         private string? _domain;
 
-        private const bool CaseSensitiveEnvironmentVariables = false;
```

Comment:
Nit: would prefer to keep this in `ProcessStartInfo.cs`:

`private bool CaseSensitiveEnvironmentVariables = !OperatingSystem.IsWindows()`;

Reply:
Actually, probably not that important come to think of it.

Reply:
Personally, I don't like the `const` because it requires us to use an ugly `#pragma warning disable`:

https://github.com/dotnet/runtime/blob/ef2a1878793e7e3fc3060396d3d2655ac53b1316/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs#L84-L88

Reply:
You could mark it `readonly` instead.